src/sonic-yang-mgmt/tests/libyang-python-tests/sample-yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,274 @@
+module sonic-acl {
```

Comment:
Did you re-update this PR upon merge of the earlier PR #3730 ?
This file already pushed into sonic-buildimage.

Reply:
As mentioned, I need to cherry-pick a fix that changes the file name to test-sonic-acl.yang. Sorry for the confusion.  Note: This PR does not have any Real YANG Models, these are just for testing our Library.

Reply:
Resolved now.